web/sites/default/config/field.field.node.location.field_geolocation.yml

```diff
@@ -15,14 +15,15 @@ third_party_settings:
     weight: 0
     geocode_field: field_address_or_entrance
     reverse_geocode_field: ''
-    hidden: false
+    skip_not_empty_value: 0
     disabled: false
+    hidden: false
     plugins:
-      - portlandmaps
+      - arcgisonline
     dumper: wkt
     delta_handling: default
     failure:
-      handling: empty
```

Comment:
@kxwang, should this be empty or preserve? It seems like preserve would be better.